lib/active_shipping/carriers/ups.rb

```diff
@@ -411,51 +452,35 @@ def build_delivery_dates_request(origin, destination, packages, pickup_date, opt
       xml_builder.to_xml
     end
 
-    def build_international_shipment_request_options(xml, origin, destination, packages, options)
-      build_location_node(xml, 'SoldTo', options[:sold_to] || destination, options)
+    def build_international_forms(xml, origin, destination, packages, options)
       if options[:paperless_invoice]
-        xml.ShipmentServiceOptions do
-          xml.InternationalForms do
-            xml.FormType('01') # 01 is "Invoice"
-            xml.InvoiceDate(options[:invoice_date] || Date.today.strftime('%Y%m%d'))
-            xml.ReasonForExport(options[:reason_for_export] || 'SALE')
-            xml.CurrencyCode(options[:currency_code] || 'USD')
-
-            if options[:terms_of_shipment]
-              xml.TermsOfShipment(options[:terms_of_shipment])
-            end
+        xml.InternationalForms do
+          xml.FormType('01') # 01 is "Invoice"
```

Comment:
Should form type be something that is passed down through the options hash? Having this hard coded doesn't feel exactly right to me.